### PR TITLE
Bump CUDSS to v0.2

### PR DESCRIPTION
**Pull request by michel2323**

### CODE REVIEW

**Comment by amontoison** [comment]
The errors are related to this issue: https://github.com/exanauts/CUDSS.jl/issues/45.
I will send an email to the CUDSS team with a MWE.

**Comment by amontoison** [comment]
The issue is fixed with CUDSS.jl v0.3.0.

File: lib/MadNLPGPU/Project.toml [hunk 1]
```diff
@@ -17,7 +17,7 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 [compat]
 AMD = "0.5"
 CUDA = "5"
-CUDSS = "0.1"
+CUDSS = "0.1, 0.2"
```
Comment (amontoison) [inline lib/MadNLPGPU/Project.toml:20]:
```suggestion
CUDSS = "0.3"
```

**Review by amontoison — CHANGES_REQUESTED** [review]

**Comment by frapac** [comment]
Superseded by #355

**Pull request closed without merging**